docs/source/monitoring-dask-cluster.rst

```diff
@@ -0,0 +1,36 @@
+Monitoring dask cluster
```

Comment:
For nowI recommend that we title this `Prometheus Monitoring` or something similarly specific.  There are many ways to monitor a Dask cluster, and if we use a more general title then we are obligated to cover all of them.

Reply:
resolved.